src/data-workspace/validation/validation-results-sidebar.module.css

```
@@ -0,0 +1,38 @@
+.wrapper {
+    margin: 16px
+}
+
+.butttons :global(button) {
+    margin-inline-end: 4px
+}
```

Consider: (also notice typo in "butttons")

```suggestion
.buttons {
    display: flex;
    gap: 4px;
}
```